templates/OperationOptionsDecodingSnippet.stencil

```diff
@@ -1,7 +1,10 @@
 {% if param.strategy == "dateFromParam" %}
 let {{ param.key }}String = DateFormatter().string(from: {{ param.key }})
 {% elif param.strategy == "byteArrayFromParam" %}
-let {{ param.key }}String = String(bytes: {{ param.key }}, encoding: .utf8)
+guard let {{ param.key }}String = String(bytes: {{ param.key }}, encoding: .utf8)?.data(using:.utf8)?.base64EncodedString() else {
+    self.options.logger.error("Failed to construct String for {{ param.key }}")
+    return
+}
```

Comment:
Why are we converting to base64EncodedString?

Reply:
Looking at generated code in other langs. In most langs, base64 encode is done in the generated code. So I have reverted this change and do the base64 encoded in the test case itself. Test cases pass with the new change.